client/src/components/budgets/BudgetsCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Card, List, Skeleton, Typography } from "antd";
+import { UserOutlined } from "@ant-design/icons/lib";
+import { Link } from "react-router-dom";
+
+import { Budget, User } from "../../generated/types";
+
+const { Title, Text } = Typography;
+
+interface Props {
+  item: Budget;
+  loading: boolean;
+}
+
+const BudgetsCard: React.FC<Props> = props => (  
+  <Link to={`/budget/${props.item.id}`}>
+    <Card
+      title={
+        <Skeleton loading={props.loading} paragraph={false} active>
+          <div className="card-head-wrapper">
+            <Title level={4} className="card-head-title">
+              {props.item.name}
+            </Title>
+          </div>
```

Comment:
Can you add some more info here about the number of categories, and the number of requisitions associated with this budget? You can use [this](https://github.com/HackGT/piranha/blob/master/client/src/util/util.ts#L136) util function and pass in the list of requisitions. You will probably have to edit the graphql query to retrieve this data. Also, you might have to edit the include on the backend. See [here](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries/#include-all-fields-for-a-specific-relation)